src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -3,7 +3,9 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics;
+#if !netcoreapp
```

Comment:
@keeratsingh, @afsanehr, @David-Engel, can you please review this change? We are trying to avoid using the Linq.Enumerable.Contains(char) API since we can use the newer/faster string.Contains(char).